src/react-image-lightbox.js

```diff
@@ -168,20 +209,46 @@ class ReactImageLightbox extends Component {
     componentWillUnmount() {
         this.mounted = false;
         this.detachListeners();
+        this.timeouts.forEach(tid => clearTimeout(tid));
+    }
+
+    setTimeout(func, time) {
+        const id = setTimeout(() => {
+            this.timeouts = this.timeouts.filter(tid => tid !== id);
+            func();
+        }, time);
+        this.timeouts.push(id);
+        return id;
+    }
+
+    clearTimeout(id) {
+        if (this.timeouts.find(tid => tid === id)) {
```

Comment:
[`Array.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Browser_compatibility) isn't supported in IE.